### PR TITLE
Lerna Upgrades

### PR DESCRIPTION
**Pull request by jordanell**
## Description

1. Standardized some packages that were shared between the two packages with the same version
2. Moved `draft-js` and `draft-convert` to peer dependencies
3. Updated `@riipen-ui/X` to `riipen-ui/X` in the `riipen-ui-docs` package (this makes it look more production ready on when displayed to the user in code examples).
4. Added `--hoist` to `bootstrap` so that similar dependencies are actually shared

## Notes

The peer dependency swap was the real fix I set off to do originally, the rest is sort of happy accidents along the way.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
+  "repository": "https://github.com/riipen/ui",
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --hoist",
     "build": "lerna run build",
     "ci:audit": "lerna run ci:audit",
     "format": "lerna run format",
-    "lint": "lerna run lint"
+    "lint": "lerna run lint",
+    "start": "lerna --scope riipen-ui-docs run start --stream"
```
Comment (jordanell) [inline package.json:17]: NOw you can start the docs site from the root of the repo.

File: packages/riipen-ui-docs/next.config.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ module.exports = {
     return map;
   },
 
-  exportTrailingSlash: true,
+  trailingSlash: true,
```
Comment (jordanell) [inline packages/riipen-ui-docs/next.config.js:33]: This was a warning message from nextjs

File: packages/riipen-ui-docs/pages/_document.jsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ export default class MyDocument extends Document {
     const { styles } = this.props;
 
     return (
-      <html lang="en">
+      <Html>
```
Comment (jordanell) [inline packages/riipen-ui-docs/pages/_document.jsx:20]: This was throwing a warning.

File: packages/riipen-ui/package.json [hunk 4]
```diff
@@ -46,6 +46,8 @@
     ]
   },
   "peerDependencies": {
+    "draft-convert": "^2.1.10",
+    "draft-js": "^0.11.6",
```
Comment (jordanell) [inline packages/riipen-ui/package.json:50]: This was the real fix I was going for. Basically we ended up having 2 version of `draft-js` and `draft-convert` in `admin-web` and `platform-web` which was causing a ton of issues.

File: packages/riipen-ui/src/components/index.js [hunk 5]
```diff
@@ -33,7 +43,15 @@ export { default as ProgressBar } from "./ProgressBar";
...
 export { default as RichTextEditor } from "./RichTextEditor";
+export { default as Spinner } from "./Spinner";
 export { default as Tab } from "./Tab";
+export { default as Table } from "./Table";
+export { default as TableBody } from "./TableBody";
+export { default as TableCell } from "./TableCell";
+export { default as TableGenerator } from "./TableGenerator";
+export { default as TableHeader } from "./TableHeader";
+export { default as TableHeaderCell } from "./TableHeaderCell";
+export { default as TableRow } from "./TableRow";
```
Comment (jordanell) [inline packages/riipen-ui/src/components/index.js:54]: Added missing exports.

File: package.json (hunk 1 above)
Comment (stredger) [reply package.json:17]: ah thats really nice actually

**Review by stredger — APPROVED** [review]
Cool beans

**Pull request merged**